extensions/markdown-language-features/src/features/previewManager.ts

```diff
@@ -160,14 +160,18 @@ export class MarkdownPreviewManager extends Disposable implements vscode.Webview
 		document: vscode.TextDocument,
 		webview: vscode.WebviewPanel
 	): Promise<void> {
+		const lineNumber = this._topmostLineMonitor.previousMDTextEditor?.document.uri.toString() === document.uri.toString() ? this._topmostLineMonitor.previousMDTextEditor?.visibleRanges[0].start.line : undefined;
```

Comment:
Instead of exposing `previousMDTextEditor`, can you hide this behind a method that gets the line for a given uri?

Reply:
`previousMDTextEditor` changed to an internal map with public functions for access.